src/SFA.DAS.ApplyService.InternalApi/Controllers/OrganisationSummaryController.cs

```diff
@@ -237,29 +237,6 @@ public async Task<IActionResult> GetTrusteesFromSubmitted(Guid applicationId)
         }
 
 
-        [HttpGet]
-        [Route("TrusteeData/CharityCommission/{applicationId}")]
-        public async Task<IActionResult> GetTrusteesFromCharityCommission(Guid applicationId)
-        {
-            _logger.LogInformation($"Retrieving Apply Data - charity commission - trustees for application {applicationId}");
-            var peopleInControl = new List<PersonInControl>();
-
-            var applyGatewayDetails = await GetApplyGatewayDetails(applicationId);
-
-            var trustees = applyGatewayDetails?.CharityCommissionDetails?.Trustees;
-
-            if (trustees != null)
-            {
-                foreach (var trustee in trustees.OrderBy(x => x.Name))
-                {
-                    var picName = trustee.Name;
-                    peopleInControl.Add(new PersonInControl { Name = picName, MonthYearOfBirth = null });
-                }
-            }
-
-            return Ok(peopleInControl);
-        }
-
```

Comment:
Hi, after discussing with Deven about how this will work, gateway is not changing functionality, so this will need to be put back in

Reply:
Done